code/modules/client/verbs/etips.dm

```diff
@@ -0,0 +1,13 @@
+/client/verb/toggle_tips()

```

Comment:
you mean in the preferences panel? That's what I want to do but I don't know how to use tgui.